### PR TITLE
Fix the authorized github user check

### PR DESCRIPTION
**Pull request by amisstea**

### CODE REVIEW

File: ansible/roles/operator-pipeline/templates/openshift/tasks/submission-validation.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ spec:
           --pr-url $(params.git_pr_url) \
           --git-username $(params.git_username) \
           --verbose \
-          --contacts $(params.contacts)
+          --contacts $(params.github_usernames)
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/submission-validation.yml:40]: If desired, I can update this kwarg name as well. Its description is accurate enough so I didn't bother to change it.

File: README.md [hunk 2]
```diff
@@ -112,7 +112,6 @@ tkn pipeline start operator-hosted-pipeline \
   --workspace name=pyxis-ssl-credentials,secret=operator-pipeline-api-certs \
   --workspace name=kubeconfig,secret=kubeconfig \
   --workspace name=github-bot-token,secret=github-bot-token \
-  --workspace name=kubeconfig,secret=kubeconfig \
```
Comment (MarcinGinszt) [inline README.md:115]: Hey! Are you sure about the removal of that from there? As it still is in the pipeline definition, comments, ansible

**Review by MarcinGinszt — APPROVED** [review]
Apart from one nitpick- LGTM, thanks for the additional, minor fixes!

File: README.md (hunk 2 above)
Comment (amisstea) [reply README.md:115]: This is just removal of a duplicate workspace also provided two lines above.

**Pull request merged**